package.json

```diff
@@ -24,6 +24,7 @@
     "body-parser": "^1.15.2",
     "config": "^1.24.0",
     "cors": "^2.8.3",
+    "express": "^4.16.2",
```

Comment:
@patrickguinness this should not be needed. `botkit` installs `express`, and the module should leverage the `express` from `botkit`. It maybe that we need to pass a reference to express into the module. I will have a look